.github/workflows/release.yml

```diff
@@ -8,6 +8,9 @@ name: Containerd Release
 env:
   GO_VERSION: '1.19.2'
 
+permissions:  # added using https://github.com/step-security/secure-workflows
+  contents: read
```

Comment:
This one probably needs write on the `release` job.

Reply:
added!